### ISSUE TITLE
Refactor: Invalid Comments/Descriptions (Missing Content Description)

### ISSUE THREAD

**Issue by Yerson-Borda** [Yerson-Borda/ScareMe#18]
**Invalid Comments/Descriptions**
file: [NavigationBar.kt](https://github.com/Yerson-Borda/ScareMe/blob/master/app/src/main/java/com/example/scareme/presentation/bottomnav/NavigationBar.kt)
```Kotlin
contentDescription = null
```
_WHY?_
Accessibility issue. Not providing a contentDescription for icons violates the accessibility principle and is flagged by Android Lint. This could be considered a missing or incorrect comment/documentation flaw.

_WHY THIS APPEARED?_
Often ignored during UI prototyping, but critical for screen reader users.
